packages/design-preview/src/components/sidebar.tsx

```diff
@@ -60,13 +66,18 @@ const Sidebar: React.FC< SidebarProps > = ( {
 	actionButtons,
 	siteId,
 	stylesheet,
+	selectedColorVariation,
+	onSelectColorVariation,
 	selectedFontVariation,
 	onSelectFontVariation,
 } ) => {
 	const translate = useTranslate();
 	const [ isShowFullDescription, setIsShowFullDescription ] = useState( false );
 	const isShowDescriptionToggle = shortDescription && description !== shortDescription;
 
+	// The variations is undefined when we're still loading the design details
+	const hasStyleVariations = ! variations || variations.length > 0;
```

Comment:
Should it be:
```suggestion
	const hasStyleVariations = variations || variations.length > 0;
```

Otherwise, `! variations` doesn't match with the words `hasStyleVariations` 🤔

Reply:
Reverted changes here as the variation is `undefined` when it's still loading, and I cannot come up with the words to represent the situation 🥲